### DISCUSSION TITLE
Does it work on unregistered sublime?

### DISCUSSION THREAD

**Discussion by ivan1993spb** [facelessuser/merge-dracula-theme#6] [Q&A] [answered]
Hi @facelessuser !

Thank you very much for this beautiful color scheme! Great job!

I really wanna use it, but I couldn't install it. No errors, no messages, no colors, Merge just completely ignores the config and that's it! I couldn't figure out what's wrong. Done everything according to your instruction.

I have Ubuntu 20, the latest **unregistered** sublime text and merge.

Does your scheme work with an unregistered copy?
Do I need to take some extra steps if it's unregistered?

**Comment by facelessuser** [comment, marked as answer]
Unregistered merge does not allow theming. You are just sick with the default light theme. I am unaware of workarounds except registering Merge.

**Reply by ivan1993spb** [reply]
Thank you for the answer!
